catalogs/sources/gtfs/realtime/us-maryland-towson-loop-gtfs-rt-s-a-1786.json

```diff
@@ -0,0 +1,13 @@
+{
+    "mdb_source_id": 1786,
+    "data_type": "gtfs-rt",
+    "entity_type": "sa",
```

Comment:
Can do! When I didn't return an array for `static_reference` it generated an error but no error was generated for `entity_type`. Can we add an issue for this?

Reply:
In both cases, the operation itself does not validate the entry. The error generated for the `static_reference` was because it is used somewhere in the code. 

Until now, the responsibility was on the user to follow the documentation (which has to be updated for this case), but we could decide to add a validation layer in the operations. 

The integration tests verify that the content of the source files is correct. Also, some may not use the operations. So we could:
1. Improve the documentation so that it is very easy to follow step by step what to do.
2. Add the validation layer in the operations.
3. Both.

Reply:
Got it. Step #2 is the most feasible for now (as in I can work on it this week). #3 would be a backlog item I can add.